### DISCUSSION TITLE
Something like this shows up after updated today.

### DISCUSSION THREAD

**Discussion by JesChen** [bmaltais/kohya_ss#907] [Q&A]
[Dataset 0]
loading image sizes.
100%|█████████████████████████████████████████████████████████████████████████████████| 23/23 [00:00<00:00, 240.19it/s]
make buckets
min_bucket_reso and max_bucket_reso are ignored if bucket_no_upscale is set, because bucket reso is defined by image size automatically / bucket_no_upscaleが指定された場合は、bucketの解像度は画像サイズから自動計算されるため、min_bucket_resoとmax_bucket_resoは無視されます
number of images (including repeats) / 各bucketの画像枚数（繰り返し回数を含む）
bucket 0: resolution (512, 512), count: 460
mean ar error (without repeats): 0.0
prepare accelerator
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ E:\Kohya\kohya_ss\train_db.py:477 in <module>                                                    │
│                                                                                                  │
│   474 │   args = parser.parse_args()                                                             │
│   475 │   args = train_util.read_config_from_file(args, parser)                                  │
│   476 │                                                                                          │
│ ❱ 477 │   train(args)                                                                            │
│   478                                                                                            │
│                                                                                                  │
│ E:\Kohya\kohya_ss\train_db.py:96 in train                                                        │
│                                                                                                  │
│    93 │   │   │   f"gradient_accumulation_stepsが{args.gradient_accumulation_steps}に設定され    │
│    94 │   │   )                                                                                  │
│    95 │                                                                                          │
│ ❱  96 │   accelerator, unwrap_model = train_util.prepare_accelerator(args)                       │
│    97 │                                                                                          │
│    98 │   # mixed precisionに対応した型を用意しておき適宜castする                                │
│    99 │   weight_dtype, save_dtype = train_util.prepare_dtype(args)                              │
│                                                                                                  │
│ E:\Kohya\kohya_ss\library\train_util.py:2975 in prepare_accelerator                              │
│                                                                                                  │
│   2972 │   │   │   if args.wandb_api_key is not None:                                            │
│   2973 │   │   │   │   wandb.login(key=args.wandb_api_key)                                       │
│   2974 │                                                                                         │
│ ❱ 2975 │   accelerator = Accelerator(                                                            │
│   2976 │   │   gradient_accumulation_steps=args.gradient_accumulation_steps,                     │
│   2977 │   │   mixed_precision=args.mixed_precision,                                             │
│   2978 │   │   log_with=log_with,                                                                │
│                                                                                                  │
│ E:\Kohya\kohya_ss\venv\lib\site-packages\accelerate\accelerator.py:355 in __init__               │
│                                                                                                  │
│    352 │   │   if self.state.mixed_precision == "fp16" and self.distributed_type != Distributed  │
│    353 │   │   │   self.native_amp = True                                                        │
│    354 │   │   │   if not torch.cuda.is_available() and not parse_flag_from_env("ACCELERATE_USE  │
│ ❱  355 │   │   │   │   raise ValueError(err.format(mode="fp16", requirement="a GPU"))            │
│    356 │   │   │   kwargs = self.scaler_handler.to_kwargs() if self.scaler_handler is not None   │
│    357 │   │   │   if self.distributed_type == DistributedType.FSDP:                             │
│    358 │   │   │   │   from torch.distributed.fsdp.sharded_grad_scaler import ShardedGradScaler  │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
ValueError: fp16 mixed precision requires a GPU
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ C:\Users\Jeslie\AppData\Local\Programs\Python\Python310\lib\runpy.py:196 in _run_module_as_main  │
│                                                                                                  │
│   193 │   main_globals = sys.modules["__main__"].__dict__                                        │
│   194 │   if alter_argv:                                                                         │
│   195 │   │   sys.argv[0] = mod_spec.origin                                                      │
│ ❱ 196 │   return _run_code(code, main_globals, None,                                             │
│   197 │   │   │   │   │    "__main__", mod_spec)                                                 │
│   198                                                                                            │
│   199 def run_module(mod_name, init_globals=None,                                                │
│                                                                                                  │
│ C:\Users\Jeslie\AppData\Local\Programs\Python\Python310\lib\runpy.py:86 in _run_code             │
│                                                                                                  │
│    83 │   │   │   │   │      __loader__ = loader,                                                │
│    84 │   │   │   │   │      __package__ = pkg_name,                                             │
│    85 │   │   │   │   │      __spec__ = mod_spec)                                                │
│ ❱  86 │   exec(code, run_globals)                                                                │
│    87 │   return run_globals                                                                     │
│    88                                                                                            │
│    89 def _run_module_code(code, init_globals=None,                                              │
│                                                                                                  │
│ in <module>:7                                                                                    │
│                                                                                                  │
│   4 from accelerate.commands.accelerate_cli import main                                          │
│   5 if __name__ == '__main__':                                                                   │
│   6 │   sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])                         │
│ ❱ 7 │   sys.exit(main())                                                                         │
│   8                                                                                              │
│                                                                                                  │
│ E:\Kohya\kohya_ss\venv\lib\site-packages\accelerate\commands\accelerate_cli.py:45 in main        │
│                                                                                                  │
│   42 │   │   exit(1)                                                                             │
│   43 │                                                                                           │
│   44 │   # Run                                                                                   │
│ ❱ 45 │   args.func(args)                                                                         │
│   46                                                                                             │
│   47                                                                                             │
│   48 if __name__ == "__main__":                                                                  │
│                                                                                                  │
│ E:\Kohya\kohya_ss\venv\lib\site-packages\accelerate\commands\launch.py:1104 in launch_command    │
│                                                                                                  │
│   1101 │   elif defaults is not None and defaults.compute_environment == ComputeEnvironment.AMA  │
│   1102 │   │   sagemaker_launcher(defaults, args)                                                │
│   1103 │   else:                                                                                 │
│ ❱ 1104 │   │   simple_launcher(args)                                                             │
│   1105                                                                                           │
│   1106                                                                                           │
│   1107 def main():                                                                               │
│                                                                                                  │
│ E:\Kohya\kohya_ss\venv\lib\site-packages\accelerate\commands\launch.py:567 in simple_launcher    │
│                                                                                                  │
│    564 │   process = subprocess.Popen(cmd, env=current_env)                                      │
│    565 │   process.wait()                                                                        │
│    566 │   if process.returncode != 0:                                                           │
│ ❱  567 │   │   raise subprocess.CalledProcessError(returncode=process.returncode, cmd=cmd)       │
│    568                                                                                           │
│    569                                                                                           │
│    570 def multi_gpu_launcher(args):                                                             │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
CalledProcessError: Command '['E:\\Kohya\\kohya_ss\\venv\\Scripts\\python.exe', 'train_db.py', '--enable_bucket',
'--pretrained_model_name_or_path=runwayml/stable-diffusion-v1-5', '--train_data_dir=E:/Kohya/Lora Training
Data/Dark/image', '--resolution=512,512', '--output_dir=E:/Kohya/Lora Training Data/Dark/model',
'--logging_dir=E:/Kohya/Lora Training Data/Dark/log', '--save_model_as=safetensors', '--output_name=Dark',
'--max_data_loader_n_workers=0', '--learning_rate=1e-05', '--lr_scheduler=cosine', '--lr_warmup_steps=46',
'--train_batch_size=1', '--max_train_steps=460', '--save_every_n_epochs=1', '--mixed_precision=fp16',
'--save_precision=fp16', '--cache_latents', '--optimizer_type=AdamW8bit', '--max_data_loader_n_workers=0',
'--bucket_reso_steps=64', '--xformers', '--bucket_no_upscale']' returned non-zero exit status 1.

**Comment by JesChen** [comment]
How can I deal with thisssss?:{{{{

**Reply by idlebg** [reply]
accelerate config - set it as in the tutorial
accelerate test - to test

you have a GPU right :D ?
run nvidia-smi
and python -m xformers.info
and paste output here

**Comment by Ar7ific1al** [comment]
Having this problem, as well. Made a fresh install and had the problem still. Tried a third fresh install, and the installation throws an error.
I've just gone and downloaded a previous release and will stick with that. Probably won't update ever again. lol

**Reply by idlebg** [reply]
can you run those and share the results
python -m xformers.info
nvcc --version
pip show torch
python -c "import tensorflow as tf; print(tf.config.list_physical_devices('GPU'))"

probably torch not installed properly..
can you share some installation logs?
